srp.go

```diff
@@ -27,7 +27,7 @@ type SrpGroup struct {
 var KnownGroups = make(map[string]*SrpGroup)
 
 func init() {
-	g3072 := &SrpGroup{g: big.NewInt(5), N: new(big.Int)}
+	g3072 := &SrpGroup{g: big.NewInt(2), N: new(big.Int)}
```

Comment:
@jpgoldberg The 3027 group is not available in [the op implementation](https://github.com/agilebits/op/blob/master/crypto/srp/srp.go) (the 4096 and 6144 and 8192 groups are). Should we be adding the 3027 group to `op`?